changelogs/unreleased/bump-ncurses-to-6.3-20220716.md

```diff
@@ -0,0 +1,3 @@
+## feature/build
+
+* Bump Ncurses from 6.2 to 6.3-20220716 for static build.
```

Comment:
Nit: We use the past tense in this context. See https://www.tarantool.io/en/doc/latest/contributing/release_notes/.